Day-II-Calculator/calculator/src/components/OperationComponents/OperationComponents.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Operation.css';
+
+function OperationButton(props){
+    return(
+        <div className={props.style}>
+            {props.text}
```

Comment:
All of your buttons look identical. Why not combine them into one component (i.e., `CalculatorButton`)?